### PR TITLE
fix stacktrace when mmdb file are not present

### LINKED ISSUE #932: Bug/Crash when geoip data files are missing
**Describe the bug**

When launching crowdsec with geoip-enriched enabled with missing geoip data, crowdsec crashes.

**To Reproduce**
1. enable geoip-enrich parser
2. remove related mmdb files
3. start crowdsec

```
version: 1.0.9-2+b4-debian
Codename: alphaga
BuildDate: 2021-08-04_20:01:32
GoVersion: 1.15.9
goroutine 115 [running]:
runtime/debug.Stack(0xc000011110, 0xc00084c780, 0x5e)
	runtime/debug/stack.go:24 +0x9f
github.com/crowdsecurity/crowdsec/pkg/types.CatchPanic(0x12942e6, 0x11)
	github.com/crowdsecurity/crowdsec/pkg/types/utils.go:100 +0x238
panic(0x11879c0, 0xc0009d0d80)
	runtime/panic.go:969 +0x1b9
github.com/crowdsecurity/crowdsec/pkg/parser.GeoIpCity(0xc00069b790, 0xe, 0xc000818640, 0x0, 0x0, 0x1, 0x1, 0x0)
	github.com/crowdsecurity/crowdsec/pkg/parser/enrich_geoip.go:82 +0x73b
github.com/crowdsecurity/crowdsec/pkg/parser.(*Node).ProcessStatics(0xc000a33268, 0xc000544480, 0x9, 0x9, 0xc000818640, 0x1, 0x1)
	github.com/crowdsecurity/crowdsec/pkg/parser/runtime.go:146 +0x4d5
github.com/crowdsecurity/crowdsec/pkg/parser.(*Node).process(0xc000a33268, 0xc000818640, 0xc000316090, 0xc000ff5c40, 0x3, 0x4, 0x1, 0xc00014f7c0, 0x17, 0x1, ...)
	github.com/crowdsecurity/crowdsec/pkg/parser/node.go:323 +0x151c
github.com/crowdsecurity/crowdsec/pkg/parser.Parse(0xc000316090, 0xc000ff5c40, 0x3, 0x4, 0x1, 0xc00014f7c0, 0x17, 0x0, 0x0, 0x0, ...)
	github.com/crowdsecurity/crowdsec/pkg/parser/runtime.go:288 +0xabc
main.runParse(0xc000d1b5c0, 0xc000d1b620, 0xc000316090, 0xc000ff5c40, 0x3, 0x4, 0x1, 0xc00014f7c0, 0x17, 0xc0006aca80, ...)
	github.com/crowdsecurity/crowdsec/cmd/crowdsec/parse.go:28 +0x2e8
main.runCrowdsec.func1.1(0x0, 0x0)
	github.com/crowdsecurity/crowdsec/cmd/crowdsec/crowdsec.go:55 +0x125
gopkg.in/tomb%2ev2.(*Tomb).run(0x19277e0, 0xc0003fd8e0)
	gopkg.in/tomb.v2/tomb.go:163 +0x38
created by gopkg.in/tomb%2ev2.(*Tomb).Go
	gopkg.in/tomb.v2/tomb.go:159 +0xbc
```

**Expected behavior**
crowdsec should gracefully stop at initialization time.

**Additional context**
Add any other context about the problem here, for example `/var/log/crowdsec.log` or error messages.

### PR DESCRIPTION
**Pull request by AlteredCoder**
fix #932

### CODE REVIEW

**Review by sabban — CHANGES_REQUESTED** [review]
looks good except for the linter error, it seems indeed that method_found is either not used or misused

File: pkg/parser/enrich.go [hunk 1]
```diff
@@ -12,94 +10,62 @@ type EnrichFunc func(string, *types.Event, interface{}) (map[string]string, erro
...
-				t = t.AddDate(time.Now().Year(), 0, 0)
-			}
-			retstr, err := t.MarshalText()
-			if err != nil {
-				log.Warningf("Failed marshaling '%v'", t)
-				continue
-			}
-			return string(retstr), t
+	for _, enricher := range EnrichersList {
+		log.Infof("Initiating enricher '%s'", enricher.Name)
```
Comment (buixor) [inline pkg/parser/enrich.go:59]: I think we either want this one or the "successfully registered ..." but having both is a bit too much

File: pkg/parser/runtime.go [hunk 2]
```diff
@@ -143,29 +143,26 @@ func (n *Node) ProcessStatics(statics []types.ExtraField, event *types.Event) er
...
-						event.Enriched[k] = v
-					}
-					break
-				} else {
-					clog.Warningf("method '%s' doesn't exist or plugin not initialized", static.Method)
+			if enricherPlugin, ok := n.EnrichFunctions.Registered[static.Method]; ok {
+				clog.Tracef("Found method '%s'", static.Method)
+				ret, err := enricherPlugin.EnrichFunc(value, event, enricherPlugin.Ctx)
+				if err != nil {
+					clog.Fatalf("plugin function error : %v", err)
```
Comment (buixor) [inline pkg/parser/runtime.go:150]: maybe fatal is too much + we want to log method name

**Review by buixor — CHANGES_REQUESTED** [review]
minor suggestions

**Review by buixor — APPROVED** [review]
lgtm

**Review by sabban — APPROVED** [review]

**Pull request merged**